2023-08-20 19:01:02,652 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:01:41,618 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:04:14,281 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:04:52,577 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:05:24,924 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework/settings.py changed, reloading.
2023-08-20 19:05:25,175 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:06:08,234 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:06:38,908 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:09:51,289 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:12:11,658 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:12:56,908 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:13:12,138 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework/settings.py changed, reloading.
2023-08-20 19:13:12,432 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:13:48,937 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:16:10,583 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-20 19:16:14,142 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 371
2023-08-20 19:16:16,893 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:17,591 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:18,091 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:18,391 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:18,624 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:18,858 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:19,057 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:19,241 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:19,442 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:19,658 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:19,841 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:20,041 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:20,242 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:20,425 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:20,625 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-20 19:16:20,791 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-20 19:16:20,975 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:03:51,469 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-21 11:03:54,507 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 371
2023-08-21 11:03:56,572 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:03:57,542 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-21 11:03:58,004 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:03:58,338 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-21 11:03:58,671 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:03:58,921 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-21 11:03:59,122 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:03:59,339 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-21 11:03:59,505 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:03:59,722 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-21 11:06:59,487 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-21 11:07:03,310 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 371
2023-08-21 11:07:05,615 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:07:06,647 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-21 11:07:07,097 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-21 11:07:07,430 - INFO - basehttp - basehttp.py - "GET /home HTTP/1.1" 200 371
2023-08-21 11:07:07,814 - INFO - basehttp - basehttp.py - "GET /about_me HTTP/1.1" 200 361
2023-08-25 15:33:13,252 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/models.py changed, reloading.
2023-08-25 15:33:13,589 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 15:33:15,729 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework/urls.py changed, reloading.
2023-08-25 15:33:16,103 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 15:50:15,556 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 15:54:25,716 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/models.py changed, reloading.
2023-08-25 15:54:26,046 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 15:55:23,167 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 15:56:34,912 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework/urls.py changed, reloading.
2023-08-25 15:56:35,236 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 15:58:13,345 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 16:01:43,581 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 16:01:46,926 - ERROR - log - log.py - Internal Server Error: /
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 14, in index
    return render(reqwest, 'Homework2/index.html', {'menu': menu, 'title': 'Главная страница'})
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 293, in do_extends
    nodelist = parser.parse()
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 226, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'content' appears more than once
2023-08-25 16:01:46,929 - ERROR - basehttp - basehttp.py - "GET / HTTP/1.1" 500 146811
2023-08-25 16:01:47,017 - WARNING - log - log.py - Not Found: /favicon.ico
2023-08-25 16:01:47,017 - WARNING - basehttp - basehttp.py - "GET /favicon.ico HTTP/1.1" 404 3081
2023-08-25 16:02:20,010 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 169
2023-08-25 16:02:39,515 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 143
2023-08-25 16:02:40,298 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 143
2023-08-25 16:02:49,534 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 634
2023-08-25 16:02:51,432 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 764
2023-08-25 16:05:17,282 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 659
2023-08-25 16:05:19,963 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1187
2023-08-25 16:05:27,837 - INFO - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 200 40
2023-08-25 16:05:35,519 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 738
2023-08-25 18:31:40,255 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1187
2023-08-25 18:31:42,732 - INFO - basehttp - basehttp.py - "GET /about HTTP/1.1" 200 629
2023-08-25 18:35:45,660 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/models.py changed, reloading.
2023-08-25 18:35:46,016 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 18:37:05,842 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/models.py changed, reloading.
2023-08-25 18:37:06,145 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:08:27,847 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 634
2023-08-25 19:08:28,593 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:08:28,944 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:08:29,644 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 738
2023-08-25 19:08:37,774 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1187
2023-08-25 19:09:22,156 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1189
2023-08-25 19:10:57,276 - ERROR - log - log.py - Internal Server Error: /create_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 46, in create_client
    return render(request, {'Добавлен новый клиент'})
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/base.py", line 17, in get_template
    for origin in self.get_template_sources(template_name):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/cached.py", line 70, in get_template_sources
    yield from loader.get_template_sources(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/filesystem.py", line 35, in get_template_sources
    name = safe_join(template_dir, template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/utils/_os.py", line 17, in safe_join
    final_path = abspath(join(base, *paths))
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/posixpath.py", line 90, in join
    genericpath._check_arg_types('join', a, *p)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/genericpath.py", line 152, in _check_arg_types
    raise TypeError(f'{funcname}() argument must be str, bytes, or '
TypeError: join() argument must be str, bytes, or os.PathLike object, not 'set'
2023-08-25 19:10:57,283 - ERROR - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 500 122159
2023-08-25 19:16:43,292 - ERROR - log - log.py - Internal Server Error: /create_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 46, in create_client
    return render(request, {'Добавлен новый клиент'})
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/base.py", line 17, in get_template
    for origin in self.get_template_sources(template_name):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/cached.py", line 70, in get_template_sources
    yield from loader.get_template_sources(template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loaders/filesystem.py", line 35, in get_template_sources
    name = safe_join(template_dir, template_name)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/utils/_os.py", line 17, in safe_join
    final_path = abspath(join(base, *paths))
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/posixpath.py", line 90, in join
    genericpath._check_arg_types('join', a, *p)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/genericpath.py", line 152, in _check_arg_types
    raise TypeError(f'{funcname}() argument must be str, bytes, or '
TypeError: join() argument must be str, bytes, or os.PathLike object, not 'set'
2023-08-25 19:16:43,293 - ERROR - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 500 122159
2023-08-25 19:17:28,618 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:17:28,991 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:17:30,154 - INFO - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 200 730
2023-08-25 19:17:33,280 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1189
2023-08-25 19:17:35,980 - ERROR - log - log.py - Internal Server Error: /create_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2053, in get_prep_value
    return int(value)
ValueError: invalid literal for int() with base 10: ''

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 44, in create_client
    new_client.save()
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1821, in execute_sql
    for sql, params in self.as_sql():
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1745, in as_sql
    value_rows = [
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1746, in <listcomp>
    [
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1747, in <listcomp>
    self.prepare_value(field, self.pre_save_val(field, obj))
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1686, in prepare_value
    return field.get_db_prep_save(value, connection=self.connection)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 954, in get_db_prep_save
    return self.get_db_prep_value(value, connection=connection, prepared=False)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2060, in get_db_prep_value
    value = super().get_db_prep_value(value, connection, prepared)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 947, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/db/models/fields/__init__.py", line 2055, in get_prep_value
    raise e.__class__(
ValueError: Field 'telefone_number' expected a number but got ''.
2023-08-25 19:17:35,984 - ERROR - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 500 139870
2023-08-25 19:18:32,839 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:18:33,181 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:20:01,089 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:20:01,444 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:23:00,136 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:23:00,499 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:23:03,290 - INFO - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 200 816
2023-08-25 19:23:06,531 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1189
2023-08-25 19:23:12,119 - INFO - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 200 816
2023-08-25 19:23:14,599 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1189
2023-08-25 19:24:06,644 - INFO - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 200 730
2023-08-25 19:24:09,216 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1189
2023-08-25 19:24:15,588 - INFO - basehttp - basehttp.py - "POST /create_client HTTP/1.1" 200 816
2023-08-25 19:24:17,986 - INFO - basehttp - basehttp.py - "GET /create_client HTTP/1.1" 200 1189
2023-08-25 19:30:04,251 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/urls.py changed, reloading.
2023-08-25 19:30:04,601 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:31:31,414 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/urls.py changed, reloading.
2023-08-25 19:31:31,720 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:31:35,842 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 634
2023-08-25 19:31:37,548 - ERROR - log - log.py - Internal Server Error: /client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/engine.py", line 161, in find_template
    raise TemplateDoesNotExist(name, tried=tried)
django.template.exceptions.TemplateDoesNotExist: Homework2/client/base.html

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 23, in client
    return render(reqwest, 'Homework2/client/client.html', {'posts': posts, 'title': 'Все клиенты'})
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 63, in render
    reraise(exc, self.backend)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/backends/django.py", line 84, in reraise
    raise new from exc
django.template.exceptions.TemplateDoesNotExist: Homework2/client/base.html
2023-08-25 19:31:37,551 - ERROR - basehttp - basehttp.py - "GET /client HTTP/1.1" 500 129972
2023-08-25 19:32:12,284 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 1213
2023-08-25 19:34:03,449 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 1213
2023-08-25 19:36:33,031 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 1299
2023-08-25 19:36:34,463 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:36:34,464 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:37:25,011 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:37:25,012 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:37:57,424 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:37:57,424 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:37:58,372 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:37:58,372 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:37:59,188 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:37:59,189 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:37:59,473 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:37:59,473 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:37:59,739 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:37:59,739 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:38:05,674 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:38:05,675 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:38:10,742 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:38:10,742 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:39:19,741 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:39:19,741 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:39:20,888 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:39:20,888 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:40:35,593 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:40:35,954 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:40:37,090 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 204, in _get_response
    self.check_response(response, callback)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 332, in check_response
    raise ValueError(
ValueError: The view Homework2app.views.read_client didn't return an HttpResponse object. It returned None instead.
2023-08-25 19:40:37,090 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 61546
2023-08-25 19:40:37,980 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 204, in _get_response
    self.check_response(response, callback)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 332, in check_response
    raise ValueError(
ValueError: The view Homework2app.views.read_client didn't return an HttpResponse object. It returned None instead.
2023-08-25 19:40:37,980 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 61546
2023-08-25 19:41:37,643 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:41:37,987 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:46:47,681 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:46:48,015 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:46:50,189 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 634
2023-08-25 19:46:52,677 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 1299
2023-08-25 19:46:53,906 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:46:53,907 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75580
2023-08-25 19:48:23,125 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:48:23,458 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:48:39,420 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:48:39,421 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75448
2023-08-25 19:49:49,436 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/urls.py changed, reloading.
2023-08-25 19:49:49,763 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:50:07,025 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/urls.py changed, reloading.
2023-08-25 19:50:07,347 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:50:08,869 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:50:08,870 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75291
2023-08-25 19:50:10,123 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:50:10,124 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75291
2023-08-25 19:51:04,256 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:51:04,257 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75291
2023-08-25 19:51:05,173 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:51:05,174 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75291
2023-08-25 19:51:11,647 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:51:11,648 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75448
2023-08-25 19:51:19,326 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 634
2023-08-25 19:51:20,942 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 1299
2023-08-25 19:51:21,727 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py", line 61, in read_client
    return render(request, 'read_client_form.html')
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: read_client_form.html
2023-08-25 19:51:21,727 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 75448
2023-08-25 19:55:07,208 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:55:07,537 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:56:59,672 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:57:00,020 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:57:01,152 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:57:01,152 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:57:01,795 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:57:01,795 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:57:20,258 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 634
2023-08-25 19:57:22,035 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 1299
2023-08-25 19:57:22,766 - ERROR - log - log.py - Internal Server Error: /read_client
Traceback (most recent call last):
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/imacson/Desktop/python_project/HomeworkDjangoPython/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: read_client() missing 1 required positional argument: 'client_id'
2023-08-25 19:57:22,767 - ERROR - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 500 56872
2023-08-25 19:59:46,697 - INFO - autoreload - autoreload.py - /Users/imacson/Desktop/python_project/HomeworkDjangoPython/Homework/Homework2app/views.py changed, reloading.
2023-08-25 19:59:47,017 - INFO - autoreload - autoreload.py - Watching for file changes with StatReloader
2023-08-25 19:59:48,377 - INFO - basehttp - basehttp.py - "GET / HTTP/1.1" 200 634
2023-08-25 19:59:51,780 - INFO - basehttp - basehttp.py - "GET /client HTTP/1.1" 200 1299
2023-08-25 19:59:52,979 - INFO - basehttp - basehttp.py - "GET /read_client HTTP/1.1" 200 1567
